scripts/make-sbom.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+cyclonedx-py --e --format json -o cyclonedx-sbom.json
```

Comment:
this line should be removed from the conversion script. Also this should probably be renamed to something like "convert-sbom.sh"